tests/FSharp.Compiler.ComponentTests/Interop/RequiredAndInitOnlyProperties.fs

```diff
@@ -459,28 +459,3 @@ let main _ =
         |> compile
         |> shouldFail
         |> withSingleDiagnostic (Error 3545, Line 7, Col 21, Line 7, Col 30, "The following required properties have to be initalized:" + Environment.NewLine + "   property RAIO.GetSet: int with get, set" + Environment.NewLine + "   property RAIO.GetInit: int with get, set")
-
-#if !NETCOREAPP
-    [<Fact(Skip = "NET472 is unsupported runtime for this kind of test.")>]
```

Comment:
This test should not be removed, I deliberately left it this way, it supposed to fail with different reason in .NET7, we should fix the reason.

Reply:
Fixed it